docs/source/setup/RabbitMQ/Troubleshooting_RMQ.rst

```diff
@@ -0,0 +1,93 @@
+.. _Troubleshooting-RMQ:
+
+
+RabbitMQ Trouble Shooting
+=========================
+
+Check the status of the federation connection
```

Comment:
Insufficient context. Some background needs to be given here as the section starts with "Check the status...". Probably James can look into this.

Reply:
This is just a FAQ kind of page so I am not sure if we need to provide context. 
Once we have the links right from the navigation bar on the left (@craig8  was going to look at it) we could probably add a line that points to the federation setup page.  What do you think?

Reply:
I'll take a stab at it after merging this PR.